utility/confirmation.go

```diff
@@ -87,3 +87,19 @@ func UserConfirmedOverwrite(resourceType string, ignoringConfirmed bool) bool {
 
 	return true
 }
+
+// ReadUserInputAndCheckIfYes is a function that can retrieve user input in form of string and checks if it is a yes.
+func ReadUserInputAndCheckIfYes(in io.Reader) error {
+	reader := bufio.NewReader(in)
+	answer, err := reader.ReadString('\n')
+	if err != nil {
+		return err
+	}
+	answer = strings.TrimRight(answer, "\r\n")
+	strings.ToLower(answer)
+	if answer != "y" && answer != "ye" && answer != "yes" {
+		return fmt.Errorf("invalid user input")
+	}
+
+	return nil
+}
```

Comment:
A couple of unit test would assure everything works fine

Reply:
I have created an issue to add test suite for the CLI - https://github.com/civo/cli/issues/271
CLI docs for object store have been updated and merged in - https://github.com/civo/cli/pull/270